qa/workunits/suites/fsx.sh

```diff
@@ -4,7 +4,6 @@ set -e
 
 git clone https://git.ceph.com/xfstests-dev.git
 cd xfstests-dev
-git checkout 12973fc04fd10d4af086901e10ffa8e48866b735
```

Comment:
We can submit another patch later, but this trivial change does fix the jobs, and it's better to wait for another version with the workload working and catching bugs than with the workload failing unconditionally

Reply:
You are right, @batrick, it would be nice to see a specific commit in such cases. However, here the severity of the approach is lower, since this is an internal ceph git repository, which I doubt will be updated often and with breaking changes.
I can't access this repository, BTW 🤷🏻

Reply:
> You are right, @batrick, it would be nice to see a specific commit in such cases. However, here the severity of the merged approach is lower, since this is an internal ceph repository, which I doubt will be updated often and with breaking changes. I can't access this repository, BTW 🤷🏻

You need to be on the sepia vpn to see it, I think.

Reply:
@lxbsz please add a followon "cleanup" PR to checkout the current HEAD commit.

Reply:
OK I found a way to look at it: https://git.ceph.com/?p=xfstests-dev.git;a=summary
It does have commit flowing, which means that other parties are using it and if anything breaks the committer will be incentivized to fix it.

If you happen to know a specific commit/tag to pin for our testing, please let me know. Otherwise, it may be even better to use the latest with all the recent tests.

Reply:
> @lxbsz please add a followon "cleanup" PR to checkout the current HEAD commit

I think we should then checkout `master`, which looks like a stable named branch. HEAD may be changed to point anywhere

Reply:
I mean whatever commit is currently `master`. Not the actual `HEAD`.

Reply:
@batrick @leonid-s-usov Sure, I will do it. Thanks

Reply:
Please see https://github.com/ceph/ceph/pull/57275.